### PR TITLE
edit README.md with how to pull request

### PR DESCRIPTION
**Pull request by Tal-Daniel**
- adds instructions for plug-in developers, how to push from their already existing repository to this repository. 

Signed-off-by: Tal Daniel <taldan@il.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,44 @@ To make your connector plug-in available to the community, submit your connector
   - Driving usage of a product or solution.
 - We also accept currency updates or bug fixes from the community for any existing connector plug-ins.
 
+### Pushing to github.com
```
Comment (IBM-klahn) [inline README.md:89]: Change the title to **Creating a pull request**

File: README.md [hunk 2]
```diff
@@ -86,6 +86,44 @@ To make your connector plug-in available to the community, submit your connector
...
+
+When your plug-in is ready for publishing and you want to contribute it to this repository, you will need to fork this repository, add your plug-in or fixes, then  create a pull request to the this repository. Instructions:
```
Comment (IBM-klahn) [inline README.md:91]: Ready to contribute?  Follow these instructions to create a pull request:

File: README.md [hunk 3]
```diff
@@ -86,6 +86,44 @@ To make your connector plug-in available to the community, submit your connector
...
+
+    ```
+    git remote add myfork https://github.com/Tal-Daniel/universal-connectors.git
+    git remote -v
+    ```
+3. Sync your repositories by running `git remote update` or `git fetch myfork`, and checkout a feature branch based on the main branch:
+    
+    git checkout -b myfork-main-fix myfork/main
+
+4. Apply your fix using a git cherry-pick, for example:
```
Comment (IBM-klahn) [inline README.md:104]: I don't understand this step. Is this a 'git add' to bring the changes from working to staging, before the commit?

**Review by IBM-klahn — COMMENTED** [review]
I made some suggestions to provide clarity around some of the steps.

**Comment by Tal-Daniel** [comment]
Thanks for the comments, Josh, I'm moving this to CONTRIBUTING.md, and will try to cram it even further. 
Closing PR until then.

File: README.md [hunk 4]
```diff
@@ -86,6 +86,44 @@ To make your connector plug-in available to the community, submit your connector
...
+    ```
+    git cherry-pick 432fbbc3
+    ``` 
+    Alternatively, if you want to edit the cherry-pick commit before it is committed, add a _--no-commit_ flag (_-n_), edit what you have to, then  commit with sign-off option (_-s_).
+    ```bash
+    git cherry-pick -n 432fbbc3
+    # edit some files, or not, then
+    git commit -s
+    ```
+    It's a good practice to sign-off *all* your commits.
```
[the remark this answers is not in the document]
Comment (ofer-haim) [reply README.md]: I think the meaning is that all of the commits will have a sign-off. Technically it's possible to have only 1 sign-off in the last commit.
We can think of a better phrasing

**Review by ofer-haim — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (Tal-Daniel) [reply README.md]: Revised it in CONTRIBUTING.md

File: README.md (hunk 3 above)
Comment (Tal-Daniel) [reply README.md]: the code snippets creates a "clean" branch from the fork, as a basis to add commits related to plugin x, or a fix, later.

**Review by IBM-klahn — APPROVED** [review]

**Pull request merged**